guides/theme-quality-assurance.md

```diff
@@ -0,0 +1,355 @@
+* When cart item quantity is changed, the update button should behave as expected
+* The remove item button should function properly
+* Active promotions should be displayed in the cart and the cart total should show their effect
+* When a promotion is active in the cart, the promotion name and discount should be displayed
+* When a promotion is unable to be used, an error message should explain why
+* Cart subtotal should display the correct subtotal amount
+* When a cart modal is part of the theme, the cart modal should display properly and fail gracefully in target browsers
+* When a cart modal is part of the theme, the cart modal should allow the user to modify items in their cart
+* When the checkout button is tapped/clicked, the user is directed to the first incomplete checkout step
+* When a user deletes all items from their cart, the cart should display no items
+* When a user attempts to proceed to checkout with an empty cart, the cart should display an error message and should not redirect to checkout
+* Tracking scripts should track the cart view correctly
+* Tracking scripts should track an item removed from the cart correctly
+* Tracking scripts should track a discount added to the cart correctly
+* The checkout call-to-action button should be prominent and should be visible without scrolling
+* Cart tap/click targets should be visible, large enough to be tapped/clicked, and unobscured by the edges of the viewport window
+
+### Shopper-Users
+{:.no_toc}
+
+* Shopper-user should be able to log in
+* Shopper-user should be able to sign up
+* Shopper-user should be able to access account
+* Shopper-user should be able to log out
+* Shopper-user should be able to edit information, email, and password
+* When a Shopper-user performs an unsuccessful sign up, log in, or account information submit attempt, they should be presented with error messages that are clear, concise, and accessible to a 5th-grade reading level
+* When a Shopper-user performs an successful sign up, log in, log out, or account information submit attempt, they should be presented with a clear success message
+* A guest shopper-user should be able create an account with a password via an emailed link after checkout
+* A Shopper-user should be able to reset their password via a password reset form
+* When a shopper-user attempts to view another user's order status, they should be shown an error
+
+### Checkout
+{:.no_toc}
+
+* Checkout view should be width-responsive and should behave as expected at phone (landscape and portrait), tablet (landscape and portrait), and desktop resolutions and aspect ratios
+* A shopper should be able to checkout on a single page
+* A shopper should be able to select from multiple methods of payment
+* When a shopper commits an error, helpful error messages should be displayed
+* When a shopper commits an error in a form field an error message should be displayed near the erroneous form field
```

Comment:
I believe that we are missing a comma in this sentence--I think it should be "When a shopper commits an error in a form field, an error message should...". 😄

Reply:
Also, I appreciate the use of "erroneous" here. 💯